tests/unit/tools/pytest_tool/test_addopts_coverage.py

```
@@ -0,0 +1,230 @@
+    Args:
+        tmp_path: Temporary directory provided by pytest.
+    """
+    (tmp_path / "tox.ini").write_text(
+        "[pytest]\naddopts = --cov=pkg\n",
+        encoding="utf-8",
+    )
+    assert_that(config_addopts_enable_coverage(tmp_path)).is_true()
+
+
+def test_banner_reports_enabled_when_addopts_force_coverage(
+    conflicting_project: Path,
```

Ruff (0.15.20)

[warning] Unused function argument: `conflicting_project`

(ARG001)